tee-worker/enclave-runtime/src/top_pool_execution.rs

```diff
@@ -271,8 +271,17 @@ where
 		)
 		.with_claim_strategy(SlotClaimStrategy::RoundRobin);
 
+	let is_single_worker = match ocall_api.get_trusted_peers_urls() {
```

Comment:
We only check if there are more workers registered, which might not really mean they are online and syncing sidechain state but that should be enough for now.

Reply:
Could you add this into code comment? :)